docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -42,7 +42,7 @@ complete clusters:
 |--------|---------------
 | Command line UX | beta
 | Config file | alpha
-| Selfhosting | alpha
+| Self-hosting | alpha
 | `kubeadm alpha` commands | alpha
 | Implementation | alpha
```

Comment:
Now beta

Reply:
We said alpha in https://github.com/kubernetes/kubernetes.github.io/pull/5497#discussion_r139277213.

Reply:
Oh, GH was showing me this comment on a weird line. We agreed that self-hosting is alpha but the overall implementation of kubeadm is now beta? I'll update that.

Reply:
What about the config file format? Also now beta?